kube-client/src/client/auth/oidc.rs

```diff
@@ -1,12 +1,6 @@
 use std::collections::HashMap;
 
-use base64::{
-    alphabet,
-    engine::{
-        fast_portable::{FastPortable, FastPortableConfig},
-        DecodePaddingMode,
-    },
-};
+use base64::{alphabet, engine, Engine as _};
```

Comment:
Getting into very nitty territory, but I think I'd prefer to refer to `base64::{alphabet,engine}` explicitly, since both of those terms are pretty ambiguous meanings.

Reply:
reasonable. done in https://github.com/kube-rs/kube/pull/1308/commits/fbf15a38204e70846a92e2aacad3371056b63349